test_scripts/Polices/build_options/016_ATF_P_TC_Timeout_Countdown_Start_PTU_Response_HTTP.lua

```diff
@@ -86,26 +91,18 @@ function Test:TestStep_Sending_PTS_to_mobile_application()
   local is_test_fail = false
   local timeout_pts
 
-
   local seconds_between_retries = {}
-  --TODO(istoimenova): Should be removed when "[GENIVI] HTTP: sdl_snapshot.json is not saved to file system" is fixed.
-  if ( commonSteps:file_exists( '/tmp/fs/mp/images/ivsu_cache/sdl_snapshot.json') ) then
-    timeout_pts = testCasesForPolicyTableSnapshot:get_data_from_PTS("module_config.timeout_after_x_seconds")
-    for i = 1, #testCasesForPolicyTableSnapshot.pts_seconds_between_retries do
-      seconds_between_retries[i] = testCasesForPolicyTableSnapshot.pts_seconds_between_retries[i].value
-    end
-  else
-    timeout_pts = testCasesForPolicyTableSnapshot:get_data_from_Preloaded_PT("module_config.timeout_after_x_seconds")
-    for i = 1, #testCasesForPolicyTableSnapshot.seconds_between_retries do
-      seconds_between_retries[i] = testCasesForPolicyTableSnapshot.seconds_between_retries[i].value
-    end
+
+  timeout_pts = testCasesForPolicyTableSnapshot:get_data_from_Preloaded_PT("module_config.timeout_after_x_seconds")
+  for i = 1, #testCasesForPolicyTableSnapshot.seconds_between_retries do
+    seconds_between_retries[i] = testCasesForPolicyTableSnapshot.seconds_between_retries[i].value
   end
 
   local time_wait = (timeout_pts*seconds_between_retries[1]*1000 + 2000)
   commonTestCases:DelayedExp(time_wait) -- tolerance 10 sec
```

Comment:
@istoimenova, Please correct comment for tolerance to 2 sec. and add some print in console that test step will last some time